### PR TITLE
Copy

### PR DESCRIPTION
**Pull request by isnamdnazim**
Please Review the code and bug me

### CODE REVIEW

File: src/Components/Button/AddNewTodoBtn.jsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const AddNewTodoBtn = () => {
   // console.log(startDate);
   // console.log(startTime);
   // disablePastTime();
-
+  //review
   const handleOk = (e) => {
     e.preventDefault();
     handleAddNewTodo(title, note, startDate, endDate, startTime, endTime);
```
Comment (tanmoypaul1005) [inline src/Components/Button/AddNewTodoBtn.jsx]: handleAddNewTodo function  after being successful modal close

File: src/Components/Icons/EditandDelete.jsx [hunk 2]
```diff
@@ -94,6 +94,7 @@ const EditandDelete = (props) => {
       },
     });
     //console.log("working");
+    //review
```
Comment (tanmoypaul1005) [inline src/Components/Icons/EditandDelete.jsx:92]: Common Input Component use

File: src/app/store.js [hunk 3]
```diff
@@ -20,6 +20,6 @@ export const getAllTodo = async () => {
     console.error(error);
   }
 };
-getAllTodo();
+//getAllTodo();
```
Comment (tanmoypaul1005) [inline src/app/store.js:23]: response using if else Statement

File: src/Services/Api/TodoApi.js [hunk 4]
```diff
@@ -114,5 +114,6 @@ export const handleCompleteTask = (id) => {
       return;
     });
 };
+//review
```
Comment (tanmoypaul1005) [inline src/Services/Api/TodoApi.js:143]: line number 27
setToDos(alltask) houer por setLoad(true) false hby

File: src/Components/Todo/IndividualTodo.jsx [hunk 5]
```diff
@@ -52,7 +52,7 @@ const IndividualTodo = () => {
         time: end,
       };
     });
-    console.log("Check Time", checkTime);
+    //console.log("Check Time", checkTime);
```
Comment (tanmoypaul1005) [inline src/Components/Todo/IndividualTodo.jsx:56]: sob input fild ka 1 ta Component kora use korban @isnamdnazim 

and ? use

File: src/app/store.js (hunk 3 above)
Comment (tanmoypaul1005) [inline src/app/store.js:23]: getAllTodo ai file a call hby na @isnamdnazim

File: src/Services/Api/TodoApi.js (hunk 4 above)
Comment (tanmoypaul1005) [inline src/Services/Api/TodoApi.js:143]: res?.data.succes status na paila tkn ki hby oita else a dea diban @isnamdnazim

File: src/Components/Button/AddNewTodoBtn.jsx [hunk 6]
```diff
@@ -49,7 +49,7 @@ const AddNewTodoBtn = () => {
   // console.log(startDate);
   // console.log(startTime);
   // disablePastTime();
-
+  //review
```
Comment (tanmoypaul1005) [inline src/Components/Button/AddNewTodoBtn.jsx]: unnecessary code remove

File: src/Components/Icons/EditandDelete.jsx [hunk 7]
```diff
@@ -94,6 +94,7 @@ const EditandDelete = (props) => {
       },
     });
     //console.log("working");
```
Comment (tanmoypaul1005) [inline src/Components/Icons/EditandDelete.jsx:91]: line number 89 handleDeleteTodo(id); fucation successful hoila todoDeletedtoast() call hby .

File: src/Components/Icons/EditandDelete.jsx (hunk 2 above)
Comment (isnamdnazim) [reply src/Components/Icons/EditandDelete.jsx:92]: okay I will make a component for common input

File: src/Services/Api/TodoApi.js (hunk 4 above)
Comment (isnamdnazim) [reply src/Services/Api/TodoApi.js:143]: done

File: src/app/store.js (hunk 3 above)
Comment (isnamdnazim) [reply src/app/store.js:23]: done

File: src/Components/Todo/IndividualTodo.jsx (hunk 5 above)
Comment (isnamdnazim) [reply src/Components/Todo/IndividualTodo.jsx:56]: okay copy

Comment (isnamdnazim) [reply src/Components/Todo/IndividualTodo.jsx:56]: Done

File: src/Components/Button/AddNewTodoBtn.jsx (hunk 6 above)
Comment (isnamdnazim) [reply src/Components/Button/AddNewTodoBtn.jsx]: done

File: src/Components/Button/AddNewTodoBtn.jsx (hunk 1 above)
Comment (isnamdnazim) [reply src/Components/Button/AddNewTodoBtn.jsx]: done

File: src/Components/Icons/EditandDelete.jsx (hunk 7 above)
Comment (isnamdnazim) [reply src/Components/Icons/EditandDelete.jsx:91]: solved

File: src/Services/Api/TodoApi.js (hunk 4 above)
Comment (isnamdnazim) [reply src/Services/Api/TodoApi.js:143]: Done Please check

**Pull request closed without merging**